tools/harness-automation/autothreadharness/settings_sample.py

```diff
@@ -92,6 +101,33 @@
 SHIELD_SIMULATION = False
 """bool: whether to simulate RF shield by changing channel"""
 
+SHIELD_CASE_ORDER = {
+    'Leader_9_2_9': [('Router_2', False), ('Commissioner', True), ('Router_1', False), ('DUT', True)],
```

Comment:
can this be declared in each test case?

Reply:
Yes,  case device drag order can be set in each case.py. From my original intention, it is more obvious, configurable and manageable in the setting file, especially when the role name (e.g. Router_2) changes in TopologyConfig.txt. What do you think?